integration-tests/src/benchmarks/cold_start_medium.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 Golem Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use async_trait::async_trait;
+
+use golem_test_framework::config::{CliParams, TestDependencies};
+use golem_test_framework::dsl::benchmark::{Benchmark, BenchmarkRecorder};
+use integration_tests::{run_benchmark, run_echo, setup, Context};
+
+struct ColdStartEchoMedium {
+    config: CliParams,
+}
+
+#[async_trait]
+impl Benchmark for ColdStartEchoMedium {
+    type IterationContext = Context;
+
+    fn name() -> &'static str {
+        "cold-start-medium"
+    }
+
+    async fn create(config: CliParams) -> Self {
+        Self { config }
+    }
+
+    async fn setup_iteration(&self) -> Self::IterationContext {
+        setup(self.config.clone(), "js-1").await
```

Comment:
The `js-1` test component does not export the `golem:it/api/echo` that your `run_echo` function is calling. Let's create new dedicated test components for this (`rust-echo`, `js-echo`, `py-echo`) with the same WIT interface. *(It can be even simpler than the `option-service` component - no need for `option` in the API just a string -> string function)

Reply:
Added the echo component in 3 different languages. The bigger wasm files (> 4Mb) still cannot be loaded(with the changes from main branch).